terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -186,56 +187,66 @@ module "glue_reporting_hub_cdc_job" {
   }
 }
 
-# Glue Job, Domain Refresh
-module "glue_domain_refresh_job" {
+# Glue Job, Create Hive Tables
+module "glue_hive_table_creation_job" {
```

Comment:
Glue job to create HIve tables in the raw_archive, structured and curated databases